ux-of-data-fetching/Exercise1/solution/src/App.tsx

```diff
@@ -37,7 +37,7 @@ function App() {
   function fetchData(url: string) {
     setLoading(true);
     setError(undefined);
-    fetch(url)
+    fetch(`${window.location.origin}${url}`)
```

Comment:
I ran into a `Failed to parse URL` error and I wasn’t able to figure out how to fix it without changing the code. 😕 It may be related to this issue: https://github.com/mswjs/msw/issues/1625